ITKImageProcessingFilters/ImportAxioVisionV4Montage.cpp

```diff
@@ -328,6 +336,43 @@ void ImportAxioVisionV4Montage::dataCheck()
     generateCache(root);
   }
 
+  if(m_MontageStart[0] > m_MontageEnd[0])
```

Comment:
Added complete sanity checking with verbose error messages to tell the user exactly what went wrong and why the preflight is failing.